src/core/dfn.js

```diff
@@ -32,3 +57,28 @@ export function run(conf) {
       }, $dfn);
   });
 }
+
+function autoPluralizeDfns() {
+  const links = new Set();
+  document
+    .querySelectorAll("a:not([href])")
```

Comment:
Make sure you test for this in the tests. I didn't see a test case that covered this.